python/smashbox/utilities/__init__.py

```diff
@@ -570,7 +570,7 @@ def share_file_with_user(filename, sharer, sharee, **kwargs):
         # TODO: this code is not the best - the goal is to trap a share not allowed error and return that error code
 
         logger.info('Share failed with %s', str(err))
-        if "not allowed to share" in str(err):
+        if 'HTTP error: 403' == err.message:
```

Comment:
@PVince81 this is actually "bad" behaviour form thy pyoc client:
https://github.com/owncloud/core/commit/7f83f2a8f2e6e3f280c3562ebdbeae9331ab5f01

The pyoc client only returns something useful for status 200:
https://github.com/owncloud/pyocclient/blob/master/owncloud/owncloud.py#L828-L837
but the ResponseError kills the actual message and only allows access to the status code:
https://github.com/owncloud/pyocclient/blob/master/owncloud/owncloud.py#L20-L28

Would be nice if you could bring the message back